lib/common/error_private.c

```diff
@@ -53,6 +53,7 @@ const char* ERR_getErrorString(ERR_enum code)
     case PREFIX(dstBuffer_wrong): return "Destination buffer is wrong";
     case PREFIX(srcBuffer_wrong): return "Source buffer is wrong";
     case PREFIX(externalMatchFinder_failed): return "External matchfinder returned an error code";
+    case PREFIX(invalid_external_sequences): return "External sequences are not valid";
```

Comment:
minor: even error codes have a naming scheme!
Looking at existing ones, the pattern is typically `topic_qualifier`.

For this case, it would be something like : 
`externalSequences_invalid`.